spec/index.bs

```diff
@@ -9609,25 +9609,36 @@ dictionary GPUCommandEncoderDescriptor
                 1. Let the {{GPUCommandBuffer/[[renderState]]}} of the currently executing
                     {{GPUCommandBuffer}} be a new [=RenderState=].
 
-                1. Let {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[colorAttachments]]}} be
+                1. Set {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[colorAttachments]]}} to
                     |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}.
-                1. Let {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[depthStencilAttachment]]}} be
+                1. Set {{GPUCommandBuffer/[[renderState]]}}.{{RenderState/[[depthStencilAttachment]]}} to
                     |descriptor|.{{GPURenderPassDescriptor/depthStencilAttachment}}.
 
                 1. For each non-`null` |colorAttachment| in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}:
-                    1. Let |colorSubresource| be |colorAttachment|.{{GPURenderPassColorAttachment/depthSlice}} of
-                        |colorAttachment|.{{GPURenderPassColorAttachment/view}}.
+                    1. Let |colorView| be |colorAttachment|.{{GPURenderPassColorAttachment/view}}.
+                    1. If |colorView|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/dimension}} is:
+                        <dl class=switch>
+                            : {{GPUTextureViewDimension/"3d"}}
+                            ::
+                                Let |colorSubregion| be |colorAttachment|.{{GPURenderPassColorAttachment/depthSlice}} of
+                                |colorView|.
+
+                            : Otherwise
+                            ::
+                                Let |colorSubregion| be |colorView|.
+                        </dl>
```

Comment:
Maybe this could broken out into an algorithm like "texture subregion for color attachment" for readability, otherwise LGTM

Reply:
I don't find it to be too problematic, so merging as-is. Can move to a separate algorithm in a follow-up if you feel stringly about it.